exporter/awsxrayexporter/translator/aws.go

```diff
@@ -161,8 +160,9 @@ func makeAws(attributes map[string]string, resource *resourcepb.Resource) (map[s
 			deployNum = 0
 		}
 		ebs = &BeanstalkMetadata{
-			Environment:  namespace,
-			VersionLabel: ver,
+			Environment: namespace,
+			// TODO(anuraaga): Implement VersionLabel
+			// VersionLabel: ver,
```

Comment:
The value wasn't set, the variable was just declared. This confused me and I think it's clearer to have this comment and not set the field until we properly implement it instead of always setting it to an empty value.

Reply:
Please do not introduce commented code. Delete it if it is not needed.

Reply:
Sure removed